contracts/metatx/ERC2771Forwarder.sol

```diff
@@ -251,11 +255,19 @@ contract ERC2771Forwarder is EIP712, Nonces {
             // Nonce should be used before the call to prevent reusing by reentrancy
             uint256 currentNonce = _useNonce(signer);
 
-            (success, ) = request.to.call{gas: request.gas, value: request.value}(
-                abi.encodePacked(request.data, request.from)
-            );
+            uint256 reqGas = request.gas;
+            address to = request.to;
+            uint256 value = request.value;
+            bytes memory data = abi.encodePacked(request.data, request.from);
```

Comment:
We can remove this memory allocation by modifying the `request.data` value in place using assembly. I think it's not a priority rn but we may open an issue.

Reply:
This idea is wrong, `request` is in `calldata` so there's no way to reuse memory.

Also, we discussed a bit and we think it's a bit kamikaze to do this memory manipulation.

Solving.